x-pack/plugins/observability_solution/observability/public/hooks/slo/use_clone_slo.ts

```diff
@@ -11,22 +11,23 @@ import { SLOWithSummaryResponse } from '@kbn/slo-schema';
 import { paths } from '../../../common/locators/paths';
 import { useKibana } from '../../utils/kibana_react';
 
-export function useCloneSlo() {
+export function useCloneSlo(remoteKibanaUrl?: string) {
```

Comment:
Since we don't actually clone the SLO this will work becuse it will redirect the browser to the "Create SLO" page on the remote Kibana but prefilled with the cloned SLO's details.